acl/src/main/java/org/apache/rocketmq/acl/AccessValidator.java

```diff
@@ -35,4 +37,32 @@
      * @param accessResource
      */
     void validate(AccessResource accessResource);
+
+    /**
+     * update the access resource config
```

Comment:
The first letter needs to be capitalized. The first letter needs to be capitalized. The first letter needs to be capitalized. Three times make help to remember.